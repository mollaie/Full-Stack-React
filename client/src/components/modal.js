const Modal = ({ handleCancel, handleOk, message }) => {
  return (
    <div className="modal-main">
      <div className="modal-header">
        <p>Delete Reservation</p>
        <button type="button" onClick={handleCancel} className="close">
          X
        </button>
      </div>
      <div className="modal-body">
        <p>{message}</p>
      </div>
      <div className="modal-footer">
        <button
          type="button"
          className="btn btn-secondary"
          onClick={handleCancel}
        >
          Close
        </button>
        <button type="button" className="btn btn-primary" onClick={handleOk}>
          Ok
        </button>
      </div>
    </div>
  );
};

export default Modal;
