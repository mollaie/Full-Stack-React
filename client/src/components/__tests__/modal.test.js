import React from "react";
import renderer from "react-test-renderer";
import modal from "../modal";

describe("modal test", () => {
  it("modal should match snapshot", () => {
    const component = renderer.create(<modal />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
