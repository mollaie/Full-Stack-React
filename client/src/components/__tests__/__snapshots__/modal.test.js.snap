// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`modal test modal should match snapshot 1`] = `<modal />`;
